spec.bs

```diff
@@ -1211,18 +1267,26 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
   };
 
   dictionary FenceEvent {
-    required DOMString eventType;
-    required DOMString eventData;
-    required sequence&lt;FenceReportingDestination&gt; destination;
+    // This dictionary has two mutually exclusive modes:
+    //
+    // When reporting to a preregistered destination (specified by enum), the following properties
```

Comment:
Can you add tighter line breaks here to accommodate the narrow IDL viewing window in the spec preview? (Same with below).